test/gcs.js

```diff
@@ -165,37 +165,36 @@ describe('UploadFS GCS', function () {
     uploadfs.copyImageIn('test.jpg', imgDstPath, (e, info) => {
       assert(!e, 'gcs copyImageIn works');
 
-      setTimeout(() => {
```

Comment:
We don't need this, modern Google Cloud Storage has read-after-write consistency.

Reply:
Easiest to read this PR with `?w=1` at the end